js/script.js

```diff
@@ -8,3 +8,121 @@ document.querySelectorAll('.navbar__menu__link').forEach((n) => n.addEventListen
+/** ***********************************8 */
+const cards = [
+  {
+    title: 'Tonic',
+    info: ['facebook', 'backend', '2015'],
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s. <span class='description-3'> Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt animi consequuntur consectetur voluptate accusantium facilis, fuga non minus, </span>",
+    skills: ['css', 'html', 'javascript'],
+    image: 'images/desktop/tonic-project-desktop.png',
+    liveLink: 'see live',
+    sourceLink: 'see source',
+  },
+  {
+    title: 'Multi-Post Stories',
+    info: ['facebook', 'full stack dev', '2015'],
+    description:
+    "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s. <span class='description-3'> Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt animi consequuntur consectetur voluptate accusantium facilis, fuga non minus, </span>",
+    skills: ['html', 'Ruby on rails', 'css', 'javascript'],
+    image: 'images/desktop/multi-post-stories-project.png',
+    liveLink: 'see live',
+    sourceLink: 'see source',
+  },
+  {
+    title: 'Facebook 360',
+    info: ['facebook', 'backend', '2015'],
+    description:
+    "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s. <span class='description-3'> Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt animi consequuntur consectetur voluptate accusantium facilis, fuga non minus, </span>",
+    skills: ['html', 'Ruby on rails', 'css', 'javascript'],
+    image: 'images/desktop/facebook-360-project.png',
+    liveLink: 'see live',
+    sourceLink: 'see source',
+  },
+  {
+    title: 'Uber Navigation',
+    info: ['UBER', ' Lead Developer', '2015'],
+    description:
+    "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s. <span class='description-3'> Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt animi consequuntur consectetur voluptate accusantium facilis, fuga non minus, </span>",
+    skills: ['html', 'Ruby on rails', 'css', 'javascript'],
+    image: 'images/desktop/uber-navigation-project.png',
+    liveLink: 'see live',
+    sourceLink: 'see source',
+  },
+];
+
+const btns = document.querySelectorAll('.works button');
+const bgModal = document.querySelector('.bg-modal');
+
+function popUp(index) {
+  const {
+    title, info, description, skills, image,
+  } = cards[index];
+  const [info1, info2, info3] = info;
+  // const [skill1, skill2, skill3] = skills;
+  // <li class="highlight font-1">${skill1}</li>
+  //   <li class="highlight font-2">${skill2}</li>
+  // <li class="highlight font-3">${skill3}</li>
```

Comment:
- [ ] Please take care of cleaning your code by removing all commented piece  of codes, this will make your work more readable and professional

Reply:
@joskalenda Please be informed that I have cleaned the code.